engineering_tools/carma

```diff
@@ -253,7 +310,20 @@ carma-config__status() {
 }
 
 carma__exec() {
-    local TARGET_IMAGE="usdotfhwastol/carma-platform:latest"
+    if [ "$1" = "-d" ]; then
```

Comment:
I think this functionality already exists in the form of the `-i` implementation below.

Reply:
I just hesitate to have redundant options for accomplishing the same outcome for clarity's sake, and this is a case (specifically overriding the currently installed carma version to exec a program) that's advanced enough the explicitness of -i fits better I feel.